src/App.js

```diff
@@ -1,16 +1,41 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+// import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
+import {useState} from 'react'
 
 const App = () => {
+  
+  const[textMessage, setTextMessage] = useState(chatMessages)
+  const likedMessage = (id) => {
+  const isliked = textMessage.map((text) =>{
+    if (text.id === id){
+      return {...text, liked: !text.liked}
+    } else{
+      return text;
+    }
+  })
```

Comment:
Nicely done! We should have this logic to change the text.liked value in App.js since the App component is already in charge of the state of textMessages.